### PR TITLE
Added a PR created date

### PR DESCRIPTION
**Pull request by coderjay12**
Fixes issue #[ #184 ]

Changes: [PR created date was published.]

Screenshots for the change:
image

Previously
image

### CODE REVIEW

File: views/partials/prs.hbs [hunk 1]
```diff
@@ -40,7 +40,7 @@
                             submitted a pull request
                             <a class="blue link underline-hover" href="{{ this.url }}">{{ this.repo_name }}#{{ this.number }}</a>
                         </div>
-                        <div class="light-silver">{{ this.title }}</div>
+                        <div class="light-silver">{{ this.title }} at {{ this.created_at }}</div>
```
Comment (jenkoian) [inline views/partials/prs.hbs]: As we're showing just the date and not the time, can we change 'at' to 'on' ?

**Comment by jenkoian** [comment]
Can you run yarn and commit the updated lock file?

File: views/partials/prs.hbs (hunk 1 above)
Comment (coderjay12) [reply views/partials/prs.hbs]: Sure yea i guess 'on' looks much better.

**Comment by jenkoian** [comment]
Thanks @coderjay12 !

**Comment by coderjay12** [comment]
Thanks to you too @jenkoian :+1:

**Pull request merged**